### DISCUSSION TITLE
Support for additional annotations?

### DISCUSSION THREAD

**Discussion by mil-ad** [ahrm/sioyek#682] [General]
I was wondering if there are any plans for implementing other types of PDF annotations other than text highlights? Freeform drawing probably doesn't make sense but perhaps text notes (sticky notes?) wouldn't be too hard? Would you be open to PRs on this?

**Comment by ahrm** [comment]
We already have an extension that does sticky notes. I don't think I will add it to the main sioyek for now. Currently I am focused on porting sioyek to android/ios so I don't plan to add major features until that is completed.

**Comment by chiavez** [comment]
I also believe more annotation tools would be nice, especially an underlining tool. I would switch to this reader if it had that. Everything else like boxes, arrows and pens would be a plus. 

Regardless, thank you for developing this nice pdf viewer. The smart jump feature is a killer.
